build.gradle

```diff
@@ -1,16 +1,25 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
-    ext.lifecycle_version = "2.2.0"
-    ext.timber_version = "4.7.1"
-    ext.nav_version = "2.3.0"
-    ext.fragment_version = "1.2.5"
+    //sdk versions
+    ext.sdk_min_version = 23
```

Comment:
As per previous comment, I think we can leave these constants out and just included them in the `build.gradle ` block below.

Reply:
fixed